src/components/app/ProfileSharing.css

```diff
@@ -92,6 +97,14 @@
   transform: translateY(24px);
 }
 
+.profileSharingCompositeButtonContainer.currentButtonIsPermalinkButton.currentButtonIsSecondShareButton .profileSharingPermalinkButton {
```

Comment:
Can we simplify the block L100-107? 
https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity